<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sistema de Archivos Distribuido</string>
  </property>
  <widget class="QListWidget" name="fileList">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>291</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMkdir">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>161</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Crear Directorio</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTransfer">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>161</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Transferir Archivo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelete">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>161</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HistorialBtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>410</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Historial Operaciones</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PeersBtn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>410</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Peers Conectados</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineaEdit1">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>180</y>
     <width>191</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>310</y>
     <width>211</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="RegistrosList">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>460</y>
     <width>560</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>280</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>RUTA:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>150</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>NOMBRE DEL DIRECTORIO:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
